client/src/components/RelatedProducts/Cards/CarouselR.jsx

```diff
@@ -25,23 +26,26 @@ function CarouselRelated({ zippedArray }) {
     }
   };
 
+  useEffect(() => {
+    setViewIndex(0);
+  }, [informationArray]);
+
   if (displayed) {
     return (
       <CarouselContainerR>
-        {zippedArray.length > 3 ? <LeftArrowR onClick={() => prev()}><FiChevronLeft size={40} /></LeftArrowR> : <LeftArrowR></LeftArrowR>}
+        {viewIndex !== 0 ? <LeftArrowR onClick={() => prev()}><FiChevronLeft size={40} /></LeftArrowR> : <LeftArrowR />}
```

Comment:
Yay! The arrows are working!!! I have some CSS notes to show you how we can move the arrows so they don't cover up the images, but this is fab!